Tests/scripts/validate_files_structure.py

```diff
@@ -682,6 +703,222 @@ def str2bool(v):
         raise argparse.ArgumentTypeError('Boolean value expected.')
 
 
+def is_text_file(file_path):
+    file_extension = os.path.splitext(file_path)[1]
+    text_file_types = set(['.yml', '.py', '.json', '.md', '.txt', '.sh', '.ini', '.eml', '', '.csv'])
+    if file_extension in text_file_types:
+            return True
+    return False
+
+
+def get_diff_text_files(files_string):
+    accepted_file_statuses = ['M', 'A']
+    all_files = files_string.split('\n')
+    text_files_list = set([])
+    for file_name in all_files:
+        file_data = file_name.split()
+        if not file_data:
+            continue
+
+        file_status = file_data[0]
+        file_path = file_data[1]
+
+        if file_status.upper() in accepted_file_statuses and is_text_file(file_path) and SECRETS_WHITE_LIST_FILE not in file_path:
+            text_files_list.add(file_path)
+
+    return text_files_list
+
+
+def get_all_diff_text_files(branch_name, is_circle):
+
+    if is_circle:
+        branch_changed_files_string = run_git_command("git diff --name-status origin/master...{}".format(branch_name))
+        text_files_list = get_diff_text_files(branch_changed_files_string)
+
+    else:
+        local_changed_files_string = run_git_command("git diff --name-status --no-merges HEAD")
+        text_files_list = get_diff_text_files(local_changed_files_string)
+
+    return text_files_list
+
+
+def validate_committed_files(branch_name, is_circle):
```

Comment:
I think you've created a duplicate in the merge, take a look at line 473

Reply:
yeah already fixed, my bad, flake8 caught it